db.go

```diff
@@ -381,7 +381,13 @@ func (db *DB) compact() (changes bool, err error) {
 		head := &rangeHead{
 			head: db.head,
 			mint: mint,
-			maxt: maxt,
+			// We remove 1 millisecond from maxt because block
+			// intervals are half-open: [b.MinTime, b.MaxTime). But
+			// chunk intervals are closed ([c.MinTime, c.MaxTime]),
```

Comment:
`([c.MinTime, c.MaxTime])` -> `[c.MinTime, c.MaxTime]` 

Let's remove brackets to not confuse here (:

Reply:
Good point, done.